### DISCUSSION TITLE
web view / crossplatform view

### DISCUSSION THREAD

**Discussion by kwo-5-18-9-14** [flxzt/rnote#881] [Q&A]
Hi, I want to know if there is a way for me to access my notes remotely. Would it be possible to view them through a web browser or on my Android phone?

**Comment by flxzt** [comment]
Currently that's not possible. One would need to write an android app to do that (see request #390)

**Reply by kwo-5-18-9-14** [reply]
I see I had a feeling but thank so much for the response I will keep update until its possible
